Vagrantfile

```diff
@@ -45,7 +67,7 @@ Vagrant.configure(2) do |config|
   # the path on the host to the actual folder. The second argument is
   # the path on the guest to mount the folder. And the optional third
   # argument is a set of non-required options.
-  config.vm.synced_folder ".", "/home/vagrant/hoot"#, group: "tomcat6"#, mount_options: ["dmode=775,fmode=775"]
+  config.vm.synced_folder ".", "/home/vagrant/hoot"#, group: "tomcat6", mount_options: ["dmode=775,fmode=775"]
```

Comment:
Does making the perms 777 remove the need to uncomment and make the "group" ownership of shared folders "tomcat6"?  If not we still want the # or even remove "mount_options".  Making everything executable causes the cmd tests to fail because they use the x mode to determine which files are tests.